docs/api/covidcast-signals/nhsn.md

```diff
@@ -31,12 +31,21 @@ For reference dates in May 2024, NHSN received data from 78% of US EDs.
 State and nation-level values are pulled directly from the source; HHS-level values are aggregated up from the state level.
 
 
-| Signal                          | Description                                                                                                                                                                         |
-|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `confirmed_admissions_covid_ew`              | Total number of patients hospitalized with confirmed COVID-19 over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Friday or Saturday of the following week. **Earliest date available:** 2020-08-08                                                                                                     |
-| `confirmed_admissions_covid_ew_prelim`          | Total number of patients hospitalized with confirmed COVID-19 over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Wednesday of the following week. <br/> **Earliest date available:** 2020-08-08  |
-| `confirmed_admissions_flu_ew`           | Total number of patients hospitalized with confirmed influenza over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Friday or Saturday of the following week.  <br/> **Earliest date available:** 2020-08-08                                                                                       |
-| `confirmed_admissions_flu_ew_prelim`     | Total number of patients hospitalized with confirmed influenza over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Wednesday of the following week. <br/> **Earliest date available:** 2020-08-08 |
+| Signal                                         | Description                                                                                                                                                                                                                                                                     |
+|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `confirmed_admissions_covid_ew`                | Total number of patients hospitalized with confirmed COVID-19 over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Friday or Saturday of the following week. **Earliest date available:** 2020-08-08                     |
```

Comment:
Some of these are missing the line breaks before "Earliest date available".  They should all be consistent.
```suggestion
| `confirmed_admissions_covid_ew`                | Total number of patients hospitalized with confirmed COVID-19 over the entire week (Sunday-Saturday). Only includes hospitalizations whose report was received before the Friday or Saturday of the following week. <br/> **Earliest date available:** 2020-08-08                     |
```